### ISSUE TITLE
Forestry Farms can crash on the Moon

### ISSUE THREAD

**Issue by ryanhaynes01** [xJon/Tekkit-2#45]
**Basic details**
- **Are you using a legitimate Minecraft account?**:  Yes
- **Are you using Java 8, 64-bit?**:  Yes
- **Did you add any additional mods**: No
- **Crash/latest log link (if applicable)**: https://mclo.gs/9CmcoFx

**Help needed**

I would like to keep my world but I literally cannot play. The second someone logs into the server, the crash occurs. Any advice is beneficial, this is also a base that has been made on the moon, if this detail provides further assistance. LV-Transformer that I used to connect some forestry farms to power caused the initial crash.

**Comment by xJon** [comment]
You can use `RestoreBackup.bat` to create a new world from the latest backup (it will be named `RestoredWorld`).
Can I ask how did you connect the Forestry farm to power? which cable?

**Comment by ryanhaynes01** [comment]
The GearBoxes were connected directly with LV RF generators, which were connected together with Glass Fibre Cables

**Comment by xJon** [comment]
Strange, will look into the issue. Either way you should be able to restore a world backup :)

**Comment by ryanhaynes01** [comment]
I just thought I'd update the issue. After I managed to get the world restore and kept resetting to world for some debugging, I found that it's an issue with using the Flux Generators. However, on Earth, this is how we managed to power the farms and everything was okay, so it might just be something to do with Flux Generators on Forestry farms in Galacticraft dimensions. This is, however, fixable using the Electric Engines directly on the farm GearBoxes! Not sure if this helps debug or anyone with a similar issue.
